ui/raidboss/data/03-hw/dungeon/gubal_library_hard.js

```diff
@@ -0,0 +1,300 @@
+'use strict';
+
+// The Great Gubal Library--Hard
+[{
+  zoneRegex: /Great Gubal Library \(Hard\)/,
+  timelineFile: 'gubal_library_hard.txt',
+  timelineTriggers: [
+    {
+      id: 'Gubal Hard Triclip',
+      regex: /Triclip/,
+      beforeSeconds: 5,
+      condition: function(data) {
+        return data.role == 'healer' || data.role == 'tank';
+      },
+      infoText: {
+        en: 'Tank Buster',
+      },
+    },
+    {
+      id: 'Gubal Hard Searing Wind',
+      regex: /Searing Wind/,
+      beforeSeconds: 5,
+      infoText: function(data) {
+        if (data.role == 'tank') {
+          return {
+            en: 'Tank Cleave on you',
+          };
+        }
+        return {
+          en: 'Avoid tank cleave',
+        };
+      },
+    },
+    {
+      id: 'Gubal Hard Properties of Darkness',
+      regex: /Darkness \(buster\)/,
+      beforeSeconds: 5,
+      condition: function(data) {
+        return data.role == 'healer' || data.role == 'tank';
+      },
+      infoText: {
+        en: 'Tank Buster',
+      },
+    },
+  ],
+  triggers: [
+    {
+      id: 'Gubal Hard Bibliocide',
+      regex: / 14:1945:Liquid Flame starts using Bibliocide/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      infoText: {
+        en: 'AoE',
+      },
+    },
+    {
+      // Ferrofluid handling is shamelessly copied from Hades Normal
```

Comment:
This isn't really important to answer, but I'm not sure what you're referring to out of Hades Normal?

Reply:
Oh, I was feeling too lazy to actually think about how to deal with Ferrofluid, so I just went to the closest encounter where I could remember a similar situation where multiple overhead markers interact. (During Hades, there's the situation where stack and spread markers come up at the same time, so it's the same concept of "do I want to stack on the target or not?")